### DISCUSSION TITLE
Cambiar de Red en AdvantEdge

### DISCUSSION THREAD

**Discussion by Acacio98** [InterDigitalInc/AdvantEDGE#358] [Q&A] [answered]
Hi all, as we know AdvantEdge runs on a machine with an associated Ip, in my case I run AdvantEdge on my laptop connected to my home network. I have tried to connect the laptop to another network, changed in the AdvantEdge configuration the new Ip associated to my laptop, but AdvantEdge does not work, I have to install everything again. What are the steps to follow if I run the platform on a network and I change network to keep AdvantEdge working on the new network.

Thanks in advance.

**Comment by roymx** [comment, marked as answer]
Hello @Acacio98,

As you are certainly aware of, AdvantEDGE uses Kubernetes (K8s) as its underlying container orchestration platform.

What you experience is a behavior of K8s which seems to use the system address at installation time in its configuration files. What we reverted doing in our labs is setting a static IP address to prevent IP address change. But this is not practical in your current setup since the laptop IP needs to change according to the network.

Running AdvantEDGE in a VM on your system would stabilize the network addresses and has been done. It requires proper VM configuration to expose the VM on the current machine but it is how most users cope with this. As a benefit, it isolates AdvantEDGE into the VM and does not expose it externally to the laptop environment.
